index.html

```diff
@@ -0,0 +1,1209 @@
+
+                      <p>When an operation requests to read a resource, the server MUST match an Authorization allowing the <code>acl:Read</code> access privilege on the resource.</p>
+
+                      <p>When an operation requests to create a resource as a member of a container resource, the server MUST match an Authorization allowing the <code>acl:Append</code> or <code>acl:Write</code> access privilege on the container for new members.</p>
+
+                      <p>When an operation requests to update a resource, the server MUST match an Authorization allowing the <code>acl:Append</code> or <code>acl:Write</code> access privilege on the resource.</p>
+
+                      <p>When an operation requests to delete a resource, the server MUST match Authorizations allowing the <code>acl:Write</code> access privilege on the resource and the containing container.</p>
+
+                      <div class="note" id="reinstated-resource-permissions" inlist="" rel="schema:hasPart" resource="#reinstated-resource-permissions">
+                        <h5 property="schema:name"><span>Note</span>: Reinstated Resource Permissions</h5>
+                        <div datatype="rdf:HTML" property="schema:description">
+                          <p>Implementations may perform cleanup tasks such as deleting the ACL resource that is associated with a resource when the resource is deleted. As deleted resources may be replaced by new resources with the same URI, access permissions on the new resource may differ when the <cite><a href="#effective-acl-resource" rel="rdfs:seeAlso">Effective ACL Resource</a></cite> is determined.</p>
+                        </div>
+                      </div>
+
+                      <p>When an operation requests to read and write an ACL resource, the server MUST match an Authorization allowing the <code>acl:Control</code> access privilege on the resource.</p>
+
+                      <div class="note" id="http-method-access-mode-mapping" inlist="" rel="schema:hasPart" resource="#http-method-access-mode-mapping">
+                        <h5 property="schema:name"><span>Note</span>: HTTP Method and Access Mode Mapping</h5>
+                        <div datatype="rdf:HTML" property="schema:description">
+                          <p>When the target of the HTTP request is the ACL resource, the operation  can only be allowed with the <code>acl:Control</code> access mode.</p>
+
+                          <p>Having <code>acl:Control</code> does not imply that the agent has <code>acl:Read</code> or <code>acl:Write</code> access to the resource itself, just to its corresponding ACL resource. For example, an agent with control access may disable their own write access (to prevent accidental over-writing of a resource by an application), but be able to change their access levels at a later point (since they retain <code>acl:Control</code> access).</p>
```

Comment:
These two statements should be outside of the note, I think, since they address any resource access and are not connected to any particular HTTP verb.

Reply:
The first line is essentially rephrasing of the requirement just before the note:
>When an operation requests to read and write an ACL resource, the server MUST match an Authorization allowing the acl:Control access privilege on the resource.

I wasn't sure if repeating in context of the note ( #http-method-access-mode-mapping ) was needed but I did it any way for some consistency in the note. Happy to leave it out if you prefer.

The second line doesn't say anything new about acl:Control that's not already mentioned in the Access Modes section ( #acl-mode-control ) but it was included in context of the operations ( #http-method-access-mode-mapping ).

The whole note could be omitted but I figured non-normative text can probably help. WAC assumes there is mapping ( #http-interactions ):

>It is assumed that servers have a mapping to handle HTTP requests and the required Authorizations to grant operations on resources.

Reply:
OK, I guess this bears repeating, I'm :+1: